docs/authentication/custom-strategies.mdx

```diff
@@ -65,21 +65,23 @@ export const Users: CollectionConfig = {
           return {
             // Send the user with the collection slug back to authenticate,
             // or send null if no user should be authenticated
-            user: usersQuery.docs[0] ? {
-              collection: 'users',
-              ...usersQuery.docs[0],
-            } : null,
+            user: usersQuery.docs[0]
+              ? {
+                  collection: 'users',
+                  ...usersQuery.docs[0],
+                }
+              : null,
```

Comment:
Unrelated to the PR. This appeared after merge from main because apparently someone used `--no-verify`. We should run prettier in CI.